content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -61,7 +70,9 @@ wget https://kubernetes.io/examples/configmap/ui.properties -O configure-pod-con
 kubectl create configmap game-config --from-file=configure-pod-container/configmap/
 ```
 
-The above command packages each file, in this case, `game.properties` and `ui.properties` in the `configure-pod-container/configmap/` directory into the game-config ConfigMap. You can display details of the ConfigMap using the following command:
+The above command packages each file, in this case, `game.properties` and `ui.properties` in the
+`configure-pod-container/configmap/` directory into the game-config ConfigMap. You can display details of the
+ConfigMap using the following command:
```

Comment:
nit: Should `game-config` use code formatting (name of ConfigMap)?

Reply:
I think it's clearer to use italics / emphasis. Not sure what the style guide thinks but _I_ want a way for readers to tell `name`s from `kind`s.